packages/site-settings/src/setup/SetupCards.tsx

```diff
@@ -5,37 +5,30 @@ import { SetupCardAccordion } from '@moderntribe/wme-ui';
 
 const Container = styled(Box)<BoxProps>(({ theme }) => ({
 	maxWidth: theme.spacing(100),
-	margin: `${ theme.spacing(3) } auto`,
+	margin: `${ theme.spacing(5) } auto`,
 }));
 
-const CardAccordionWrapper = styled(SetupCardAccordion)(({ theme }) => ({
-	'& .MuiChip-colorError.MuiChip-filledError': {
-		backgroundColor: theme.palette.error.main,
-		color: theme.palette.error.contrastText,
-	}
-}));
+const ACCORDION_TIMEOUT = 150;
 
 const SetupCards = () => {
 	return (
 		<Container>
 			{
 				CARDS.map((card: SetupCardAccordionInterface) => {
 					return (
-						<CardAccordionWrapper
+						<SetupCardAccordion
 							key={ card.id }
 							id={ card.id }
-							header={ card.header }
-							subHeader={ card.subHeader }
-							chipBackground={ card.chipBackground }
-							chipText={ card.chipText }
-							defaultExpanded
+							header={ card.title }
+							subHeader={ card.intro }
+							TransitionProps={ { timeout: ACCORDION_TIMEOUT } }
```

Comment:
You don't need the `TransitionProps` unless you're using the <JumpNav /> to scroll-to and open the card accordions.